tests/wrappers/test_onehot_observation.py

```diff
@@ -0,0 +1,9 @@
+import gym
+
+
+def test_onehot_observation():
+    env = gym.make("Taxi-v3")
+    env = gym.wrappers.OnehotObservation(env)
+    env.reset()
```

Comment:
And relatedly to the previous comment, this could use some asserts on the actual `obs` from `obs = env.reset()`